pom.xml

```diff
@@ -61,34 +61,39 @@
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<java.version>1.8</java.version>
 		
-		<sonar.projectKey>revaturelabs_rideshare-user-service</sonar.projectKey>
-  		<sonar.organization>revaturelabs-screenforce</sonar.organization>
+		<sonar.projectKey>Batch-738-Mathew-Oberlies_RideshareBackend</sonar.projectKey>
+  		<sonar.organization>batch-738-mathew-oberlies</sonar.organization>
   		<sonar.host.url>https://sonarcloud.io</sonar.host.url>
-  		<sonar.coverage.exclusion>com/revature/beans/*.java</sonar.coverage.exclusion>
```

Comment:
We do want to keep this (although it has been renamed to models instead of beans). The idea is that the code coverage should ignore our model classes.